From 7865c1fe1e11453eca2f3e69faafc25893af28e9 Mon Sep 17 00:00:00 2001
From: Nolen Johnson <johnsonnolen@gmail.com>
Date: Sat, 9 Jun 2018 17:19:32 -0400
Subject: [PATCH 1/3] klte-common: Define Vendor security patch level

* This is needed for Trust, as we don't have a Vendor image that sets a
  valid vendor security patch level.

* Taken from the G900FXXU1CRA2_G900FOXA1CRA2_BTU Factory Image.

Change-Id: I105cc7e464e581683c6784e3aef8f7297b90aea0
---
 system.prop | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/system.prop b/system.prop
index 9722f8d..0f22bf4 100644
--- a/system.prop
+++ b/system.prop
@@ -59,5 +59,8 @@ debug.sensors=1
 # Tethering
 net.tethering.noprovisioning=true
 
+# Vendor security patch level
+ro.lineage.build.vendor_security_patch=2017-08-01
+
 # WiFi
 wifi.interface=wlan0
-- 
2.17.1

